controller/controller_server/csi_controller_server.py

```diff
@@ -392,13 +392,16 @@ def CreateSnapshot(self, request, context):
         secrets = request.secrets
         user, password, array_addresses = utils.get_array_connection_info_from_secret(secrets)
         try:
-            _, vol_id = utils.get_volume_id_info(source_volume_id)
+            _, volume_id = utils.get_volume_id_info(source_volume_id)
             array_type = detect_array_type(array_addresses)
             with get_agent(user, password, array_addresses, array_type).get_mediator() as array_mediator:
                 logger.debug(array_mediator)
                 snapshot_final_name = self._get_snapshot_final_name(request, array_mediator)
 
-                volume_name = array_mediator.get_volume_name(vol_id)
```

Comment:
if you are done with this method, please remove it from the project

Reply:
done